services/docker-compose.yml

```
@@ -113,6 +113,33 @@ services:
       - interactive_services_subnet
       - computational_services_subnet
 
+  static-webserver:
+    image: ${DOCKER_REGISTRY:-itisfoundation}/static-webserver:${DOCKER_IMAGE_TAG:-latest}
+    environment:
```

```suggestion
    init: true
    environment:
```

This ensures there is a process with PID 1 started before your process